map_gen/maps/crash_site/commands.lua

```diff
@@ -307,16 +306,24 @@ function Public.control(config)
         for m in string.gmatch(location_string, "%-?%d+") do
```

Comment:
It wasn't an issue before, but it might be now if we have surface names that are valid numbers. Appreciate that's unlikely to be a problem, more just awareness if this code was copied for other scenarios that make their own surfaces. Maybe a comment would be alright for now.
```suggestion
        for m in string.gmatch(location_string, "%-?%d+") do -- Assuming the surface name isn't a valid number.
```

Same again for when we process the coords in the strike function.